<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartVax </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(63, 123, 134);

border-radius: 15px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid #000000;
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(63, 123, 134);
border: 2px solid #000000;
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="sidentifier">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 50px;
    font-weight: bold;
    color: black;
</string>
    </property>
    <property name="text">
     <string>S'IDENTIFIER</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>721</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #6D9EA3;
border: 2px solid #000000;
border-radius: 15px;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mdp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> font-size: 24px;
 font-weight: bold;
 color: black;
background-color:rgb(63, 123, 134);</string>
     </property>
     <property name="text">
      <string>MOT DE PASSE :</string>
     </property>
    </widget>
    <widget class="QLabel" name="id">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> font-size: 24px;
 font-weight: bold;
 color: black;
background-color:rgb(63, 123, 134);</string>
     </property>
     <property name="text">
      <string>IDENTIFIANT :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 123, 134);
font: 18pt &quot;Segoe UI&quot;;
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 123, 134);
font: 18pt &quot;Segoe UI&quot;;
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Face">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>90</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(63, 123, 134);
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::CameraWeb"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_35">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid #000000;
border-radius: 15px;
</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_4</zorder>
    <zorder>mdp</zorder>
    <zorder>id</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>lineEdit_mdp</zorder>
    <zorder>lineEdit_35</zorder>
    <zorder>pushButton_Face</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_conn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-weight: bold;
background-color:rgb(63, 123, 134);
    border: 2px solid black;  /* Change border color to blue */
</string>
    </property>
    <property name="text">
     <string>Se Connecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid #000000;
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>widget</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_close</zorder>
   <zorder>sidentifier</zorder>
   <zorder>pushButton_conn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
